.github/workflows/haskell.yml

```diff
@@ -0,0 +1,57 @@
+name: Haskell CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    defaults:
+      run:
+        shell: bash
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc: ["8.6.5", "8.10.2"]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        exclude:
+          - os: windows-latest
+            ghc: 8.6.5
```

Comment:
Why do we exclude this combination but not `window-latest` + `8.10.2`?

Reply:
For some reason it didn't build.  I didn't want to look into it because we're moving to `8.10.2` and `8.6.5` support will eventually be dropped.

Reply:
I've re-included it because the build looks to be working.